### DISCUSSION TITLE
显示Login successful，但是对话后没有任何回应

### DISCUSSION THREAD

**Discussion by fronaanran2** [lss233/chatgpt-mirai-qq-bot#628] [Q&A]
![image](https://user-images.githubusercontent.com/130748220/232001907-8eb2589f-2472-4671-8a81-d6b4283104a9.png)
![IDI@8)S `1@8}B_UDWV4AG2](https://user-images.githubusercontent.com/130748220/232002185-fa3b0ee8-05d3-498e-9d1d-338050be9f1b.png)
如图所示，没有回应是哪个步骤有问题吗？

**Comment by lss233** [comment]
你要发 chatgpt for bot 的日志，你发 mirai 日志无法确定问题。
建议看看那些和你一样问题的 issue，没有日志和配置文件看不出什么问题。
